validate.go

```diff
@@ -80,17 +83,21 @@ var bundleValidateCommand = cli.Command{
 		}
 
 		hooksCheck := context.Bool("hooks")
-		bundleValidate(spec, rootfsPath, hooksCheck)
+		hostCheck := context.Bool("host-specific")
+		bundleValidate(spec, rootfsPath, hooksCheck, hostCheck)
 		logrus.Infof("Bundle validation succeeded.")
 		return nil
 	},
 }
 
-func bundleValidate(spec rspec.Spec, rootfs string, hooksCheck bool) {
+func bundleValidate(spec rspec.Spec, rootfs string, hooksCheck bool, hostCheck bool) {
 	checkMandatoryField(spec)
 	checkSemVer(spec.Version)
 	checkPlatform(spec.Platform)
```

Comment:
This can probably also get a `hostCheck` that compares `spec.Platform` against the current host (`runtime.GOOS`?).